packages/e2e-tests/fixtures/blocks/core__cover__deprecated.html

```diff
@@ -0,0 +1,3 @@
+<!-- wp:cover {"url":"https://cldup.com/uuUqE_dXzy.jpg","id":35} -->
+<div class="wp-block-cover has-background-dim" style="background-image:url(https://cldup.com/uuUqE_dXzy.jpg)"><p class="wp-block-cover-text"><strong>Cover Block</strong></p></div>
```

Comment:
Can you copy one of the existing images encoded with `data:*`. I think @jorgefilipecosta updated all urls to avoid network calls in tests or something like that.